models/app.model.js

```diff
@@ -81,3 +81,14 @@ exports.patchArticleVotes = (article_id, inc_votes) => {
     return rows[0];
   });
 };
+
+exports.deleteCommentById = (comment_id) => {
+  const query = `DELETE FROM comments WHERE comment_id = $1 RETURNING * `;
+
+  return db.query(query, [comment_id]).then(({ rows }) => {
+    if (!rows.length) {
+      return Promise.reject({ status: 404, msg: "Comment Not Found" });
+    }
```

Comment:
This makes sense - a non-existent comment can be  inferred from the db returning empty rows.